realm-annotations/src/main/java/io/realm/annotations/LinkingObjects.java

```diff
@@ -22,10 +22,15 @@
 import java.lang.annotation.Target;
 
 /**
- * Annotation for defining a reverse relationship from one class to another. This annotation can
- * only be added to a field of the type {@code RealmResults}.
+ * This annotation defines a <b>Backlink</b>, a reverse relationship from one class to another.
```

Comment:
I don't think we should start talking about Backlinks as a name in the public docs. Since we decided to use linking objects as the public name, we should use that consistently or alternatively talk about "reverse relationships" which is the term the website docs use. The old sentence was IMO fine.